### ISSUE TITLE
placemat, neco: introduce staggered bootup for test stability

### ISSUE THREAD

**Issue by morimoto-cybozu** [cybozu-go/neco#890]
## What

Neco dctest sometimes fails due to instability of VMs boot-up.
This may be mitigated by booting VMs not in the same time but with applying different delay.

## How

Modify VMs first boot in placemat to apply different delay.
Modify rebooting steps in Neco dctest to apply different delay.

## Checklist

- [ ] Finish implementation of the issue
- [ ] Test all functions
- [ ] Have enough logs to trace activities
- [ ] Notify developers of necessary actions

**Comment by dulltz** [comment]
I tried to add delay for each ipmipower-start worker-node,
and noticed that increased delay increased stability.

Reboot worker node takes about 185s, so I set the upper bound of experiment as 190s.

Results:
- 30s delay: Does not improve test stability
- 60s delay: Test stability is fairly stable
- 190s delay: Test stability is stable

190s delay make test stable, but it takes 1330s (=190s * 7 nodes).
